scss/_main.scss

```diff
@@ -78,32 +78,18 @@
 .container {
```

Comment:
With this changes my index.html page looks broken. Please, if you need to use .container, do not use mine, create your own container.